tests/MongoDB.Driver.Tests/ITestRunner.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Bindings;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Tests
+{
+    public interface ITestRunner : IDisposable
```

Comment:
I'm a little worried that this interface name is too general (it implies that all possible test runners ever might implement it).

But I'm not at all sure what a better name might be.

The only thing I can think of that might be a little bet more specific is:

    IJsonDrivenTestRunner

Reply:
Not sure, but can we merge this interface with `MongoClientJsonDrivenTestRunnerBase`?